deploy_roles.tf

```diff
@@ -12,9 +12,12 @@ module "deploy_lambda_policy" {
   source = "git::https://github.com/nationalarchives/da-terraform-modules//iam_policy"
   name   = "${local.environment_title}DPGithubActionsDeployLambdaPolicy"
   policy_string = templatefile("${path.module}/templates/iam_policy/deploy_lambda_policy.json.tpl", {
-    download_metadata_lambda_arn       = module.download_metadata_and_files_lambda.lambda_arn,
-    slack_notification_lambda_arn      = module.slack_notifications_lambda.lambda_arn
-    entity_event_generation_lambda_arn = module.entity_event_generator_lambda.lambda_arn
-    bucket_name                        = "mgmt-dp-code-deploy"
+    lambda_arns = jsonencode([
+      module.download_metadata_and_files_lambda.lambda_arn,
+      module.slack_notifications_lambda.lambda_arn,
+      module.entity_event_generator_lambda.lambda_arn,
+      module.ingest_mapper_lambda.lambda_arn
+    ])
+    bucket_name = "mgmt-dp-code-deploy"
```

Comment:
This isn't a part of your changes but is it fine that the bucket is "dp" and not "dr2"?